coverage_config_aarch64.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The aarch64 coverage test is not yet running. We can remove this file and add it only when we need it.

Reply:
Done.